var/spack/repos/builtin/packages/costa/package.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Costa(CMakePackage):
+    """
+    Distributed Communication-Optimal Matrix Transpose and Reshuffle Library
+    Based on the paper: https://arxiv.org/abs/2106.06601
+    """
+
+    maintainers = ['haampie', 'kabicm']
+    homepage = 'https://github.com/eth-cscs/COSTA'
+    url = 'https://github.com/eth-cscs/COSTA/releases/download/v2.0/COSTA-v2.0.tar.gz'
+    git = 'https://github.com/eth-cscs/COSTA.git'
+
+    # note: The default archives produced with github do not have the archives
+    #       of the submodules.
+    version('master', branch='master', submodules=True)
+    version('2.0', sha256='ef283b904b1b77b7d0de401cbdc3d7850c77368f2b57249c6eaee3017794c4b8')
+
+    variant('scalapack', default=False, description='Build with ScaLAPACK API')
+    variant('openmp', default=True, description="Build with OpenMP support")
```

Comment:
shouldn't this be passed to cmake?

Reply:
You are right! I removed the openmp variant, as it's always assumed in cmake as well. The scalapack should be passed to cmake already.

Reply:
does it seem stuck or is it still fine?